### DISCUSSION TITLE
I have a problem. It appears to me that I am missing this: ; . But that is in the tutorial. What I do?

### DISCUSSION THREAD

**Discussion by lucas11222** [Minestom/Minestom#1779] [Help]
I have a problem. It appears to me that I am missing this: ; . But that is in the tutorial. What I do?

**Comment by mworzala** [comment]
Java lines end with semicolons (`;`), you are probably missing one. I would suggest very strongly that you take some time to learn basic java concepts before trying to do something with Minestom...
